style.css

```diff
@@ -496,3 +501,605 @@ button:active {
+  }
+
+  .dt-about-me {
+    display: flex;
+    flex-direction: column;
+    width: 447px;
+    padding: 24px;
+    height: 392px;
+  }
+
+  .dt-about-me-languages {
+    display: flex;
+    padding: 24px;
+    width: 684px;
+    height: 380px;
+  }
+
+  .dt-language-types {
+    display: flex;
+    flex-direction: row;
+    width: 414px;
+  }
+
+  .about-me h2 {
+    height: 101px;
+    width: auto;
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: left;
+    color: #172b4d;
+  }
+
+  .about-me p {
+    height: 144px;
+    width: auto;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #344563;
+  }
+
+  .social-container {
+    display: flex;
+    flex-direction: column;
+    width: auto;
+    height: 80px;
+  }
+
+  .social-media1 ul {
+    display: flex;
+    justify-content: space-evenly;
+    height: 20px;
+    width: 170px;
+    margin-left: 0;
+    padding-left: 0;
+  }
+
+  .social-media1 ul li {
+    list-style: none;
+    color: #505f79;
+    margin-left: 0;
+  }
+
+  .social-media1 ul li a {
+    /* list-style: none; */
+    color: #505f79;
+  }
+
+  .resume {
+    width: fit-content;
+    background-color: #fff;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: column;
+    grid-column: auto;
+    padding-top: 24px;
+    padding-left: 0;
+  }
+
+  .lang {
+    display: flex;
+    height: 72px;
+    width: 100%;
+    justify-content: space-between;
+  }
+
+  .frame {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+    padding-bottom: 20px;
+    padding-top: 20px;
+  }
+
+  .skil {
+    display: flex;
+    width: 684px;
+    flex-grow: 1;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+    padding-bottom: 20px;
+    padding-top: 20px;
+  }
+
+  .language-type img {
+    width: 100%;
+    height: 48px;
+  }
+
+  .language-type {
+    display: flex;
+    flex-direction: column;
+    background-color: #f7f7f9;
+    border-radius: 8px;
+    color: #253858;
+    width: 414px;
+    height: 120px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin: 10px;
+    align-items: center;
+    padding: 12px;
+  }
+
+  .language-type p {
+    justify-content: center;
+    text-align: center;
+  }
+
+  /* contact-form */
+
+  .contact-form-container {
+    background-color: #6070ff;
+    width: 100%;
+    border-radius: 100px 0 0 0;
+    padding-bottom: 8px;
+    height: 846px;
+    margin-top: -256px;
+  }
+
+  .contact-form {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: center;
+    background-image: url("images/contact-form-bg-desktop.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+
+    /* background-position-x: 153px;
+    background-position-y: 21px; */
+  }
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly fix it. :writing_hand: